src/s3-resource-selector/s3-modal/basic-table.tsx

```diff
@@ -172,3 +180,39 @@ export function BasicS3Table<T>({
     />
   );
 }
+
+interface InternalHeaderActionsProps<T> {
+  loadData: () => void;
+  i18nStrings: BasicS3TableProps<T>['i18nStrings'];
+  lastUpdated: Date | undefined;
+}
+
+export function InternalHeaderActions<T>({ i18nStrings, loadData, lastUpdated }: InternalHeaderActionsProps<T>) {
+  function getLastUpdated() {
+    if (!lastUpdated || !i18nStrings.lastUpdatedText) {
+      return null;
+    }
+
+    return (
+      <InternalBox textAlign="right" color="text-status-inactive">
+        <LiveRegion visible={true}>
+          <span role="status">
```

Comment:
Ideally `<InternalBox />`, `<LiveRegion />` and `<span role="status" />` would be one element but `<LiveRegion />` doesn't receive a `role` prop.

Any suggestion?